### ISSUE TITLE
rpy2 and r-stringi support

### ISSUE THREAD

**Issue by ayaby** [jooh/neuroconda#3]
Two packages that would be useful to add are rpy2 (this worked only when I installed it from conda-forge) and r-stringi (which didn't work from conda-forge, only from r). The second is required as a dependency for quite a lot of packages, for example BayesFactor

Support for a python IDE that isn't jupyter lab would also be very useful (some basic functionalities are missing in jupyter lab)

Thanks!

**Comment by ethanknights** [comment]
I agree it might be useful at some point to add `r-stringi ` - a few extra packages I needed relied on this for me too.
Sometime, I'll have a go cloning the neurconda env & using makefile etc. to try contribute ...
